### PR TITLE
[release-1/starbucks] feat(keeper): filebeat config template support more settings

### PR DESCRIPTION
**Pull request by xpofei**
**What this PR does / why we need it**:

Add your description

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @jianglinjian 

**Release note**:

```release-note
filebeat config template support follow settings:
queue.mem:
  events:8192
output.logstash:
  worker: 4
  pipelining: 3
  bulk_max_size: 4096
```

### CODE REVIEW

File: build/filebeat-keeper/filebeat.yml.tpl [hunk 1]
```diff
@@ -2,6 +2,14 @@ http.enabled: true
...
 http.port: 5066
 
+{{- if .queueEvents }}
+queue.mem:
+  events: {{ .queueEvents }}
+{{- else }}
+queue.mem:
+  events: 8192
+{{- end }}
+
```
Comment (jianglinjian) [inline build/filebeat-keeper/filebeat.yml.tpl]:
```
queue.mem:
  events: {{ default 8192 .queueEvents }}
```
感觉这样更优雅一点🐶

**Comment by bogeit** [comment]
大佬们好，我看了下你们这个基于阿里云的log-pilot的定制版本更新挺多的，不知道现在有没有支持Containerd容器运行时呢？

**Pull request merged**